.travis.yml

```diff
@@ -26,6 +28,17 @@ jobs:
         - brew install openslide
         - echo 'export PATH="/usr/local/Frameworks/Python.framework/Versions/Current/bin:$PATH"' >> /Users/travis/.bash_profile
         - echo $PATH
+    - name: "Python: 3.8"
```

Comment:
here it's running with Python 3.7.5

Reply:
you need to specify the osx image:

I know that `xcode10.2` comes with py 3.7.3 (https://blog.travis-ci.com/2019-08-07-extensive-python-testing-on-travis-ci)

So I guess we need to find the right one between `xcode10.3` and `xcode12u`. Complete list https://docs.travis-ci.com/user/reference/osx/

Reply:
Why we need to specify the osx image if we have the proof that it's running with the right python version?

Reply:
mmmm maybe u right

Reply:
you can see the actual python version by looking at the pytest log on travis

Reply:
yes i see